.github/workflows/publish_pypi.yml

```diff
@@ -55,6 +66,8 @@ jobs:
         env:
           CIBW_ENVIRONMENT: 'PYBAMM_USE_VCPKG=ON VCPKG_ROOT_DIR=C:\vcpkg VCPKG_DEFAULT_TRIPLET=x64-windows-static-md VCPKG_FEATURE_FLAGS=manifests,registries CMAKE_GENERATOR="Visual Studio 17 2022" CMAKE_GENERATOR_PLATFORM=x64'
           CIBW_ARCHS: "AMD64"
+          CIBW_BEFORE_BUILD: python -m pip install setuptools wheel # skip CasADi and CMake
+          CIBW_TEST_COMMAND: python -c "import pybamm; pybamm.IDAKLUSolver()"
```

Comment:
I had problems with trying to use `grep` on Windows and `findstr` did not work well either, but this one works. The Python interpreter raises and exits with an error code of 1 with the trace

```python
ImportError("KLU is not installed")
```

as intended.